cli/setup.go

```diff
@@ -154,12 +154,22 @@ const (
 // ---------------------------- END Setup constants ----------------------------
 
 func getDefaultDeviceType() string {
-	hostName, err := ioutil.ReadFile("/etc/hostname")
+	var devType string
+	deviceType, err := ioutil.ReadFile(conf.DefaultDeviceTypeFile)
 	if err != nil {
```

Comment:
Also, generally speaking, I would expect the logic to represent the preference with something like:
- try reading `conf.DefaultDeviceTypeFile`
- if ok, parse and get the `devType`
- else try `/etc/hostname`
- else unknown